src/drivers/esp32/interface/esp_rom_bootloader.h

```diff
@@ -46,6 +46,15 @@
 **/
 bool espRomBootloaderSync(uint8_t *sendBuffer);
 
+/**
+* @brief Called to attach the SPI memory to the ESP. Must be called before issuing any flash command.
+*
+* @param *sendBuffer Pointer to a buffer used to construct the spi attach packet. Can be left empty.
```

Comment:
Are you supposed to have the star as part of the name here? Is it not part of the type?